msteams-platform/toolkit/TeamsFx-CLI.md

```diff
@@ -71,126 +69,90 @@ By default, `teamsfx new` goes into interactive mode and guides you through the
-```
+  ```bash
+  teamsfx new --interactive false --capabilities "notification" --bot-host-type-trigger "http-restify" --programming-language "typescript" --folder "./" --app-name       MyAppName
+  ```
 
-#### Teams app in JavaScript with tab, bot capabilities and Azure Functions
+* Teams command and response bot
 
-```bash
-teamsfx new --interactive false --app-name newtabbotapp --capabilities tab bot --programming-language javascript --azure-resources function
-```
+  ```bash
+  teamsfx new --interactive false --capabilities "command-bot" --programming-language "typescript" --folder "./" --app-name myAppName
+  ```
 
-#### Teams tab app with Azure Functions and Azure SQL
+* Tab app hosted on SPFx using React
 
-```bash
-teamsfx new --interactive false app-name newapp --azure-resources sql function --programming-language typescript
-```
+  ```bash
+  teamsfx new --interactive false --app-name newspfxapp --capabilities tab-spfx --spfx-framework-type react
+  ```
+
+## `teamsfx add`
+
+The following table lists diffetent features to your Teams application along with their descrition.
+
+| Command | Description |
+|:----------------  |:-------------|
+| `teamsfx add notification` | Send notification to Microsoft Teams through various triggers. |
+| `teamsfx add command-and-response` | Respond to simple commands in Microsoft Teams chat.|
+| `teamsfx add sso-tab` | Teams identity aware webpages embedded in Microsoft Teams.|
+| `teamsfx add tab` | Hello world webpages embedded in Microsoft Teams.|
+| `teamsfx add bot` | Hello world chatbot to run simple and repetitive tasks by user. |
+| `teamsfx add message-extension` | Hello world message extension allowing interactions through buttons and forms. |
+| `teamsfx add azure-function`| A serverless, event-driven compute solution that allows you to write less code. |
+| `teamsfx add azure-apim` | A hybrid, multicloud management platform for APIs across all environments.|
+| `teamsfx add azure-sql` | An always-up-to-date relational database service built for the cloud. |
+| `teamsfx add azure-keyvault` | A cloud service for securely storing and accessing secrets. |
+| `teamsfx add sso` | Develop a Single Sign-On feature for Teams Launch pages and Bot capability. |
+| `teamsfx add api-connection [auth-type]` | Connect to an API with authentication support using TeamsFx SDK. |
+| `teamsfx add cicd` | Add CI/CD Workflows for GitHub, Azure DevOps or Jenkins.|
 
 ## `teamsfx account`
 
-Manage cloud service accounts. The supported cloud services are `Azure` and `Microsoft 365`.
+The following table lists the cloud service accounts,such as Azure and Microsoft 365.
 
-| `teamsFx account` Command | Description |
+| Command | Description |
 |:----------------  |:-------------|
 | `teamsfx account login <service>`  | Log in to the selected cloud service. |
 | `teamsfx account logout <service>`  | log out of selected cloud service. |
```

Comment:
| `teamsfx account login <service>`  | Log in to the selected cloud service. Service options are M365 or Azure. |
| `teamsfx account logout <service>`  | log out of selected cloud service. Service options are M365 or Azure. |

Reply:
Updated as suggested.